dev/tool/src/workspace.ts

```diff
@@ -61,3 +72,40 @@ export async function diffWorkspace (mongoUrl: string, workspace: WorkspaceId, r
     await client.close()
   }
 }
+
+export async function updateField (
+  mongoUrl: string,
+  workspaceId: WorkspaceId,
+  transactorUrl: string,
+  cmd: { objectId: string, objectClass: string, type: string, attribute: string, value: string }
+): Promise<void> {
+  const connection = (await connect(transactorUrl, workspaceId, undefined, {
+    mode: 'backup'
```

Comment:
Let's remove use of domain, and pass it as parameter. Since this logic will be done before we will start any test it should be fine.